series/plzen-pyvo/events/2020-05-13.yaml

```diff
@@ -2,48 +2,42 @@ city: plzen
-  Jako odpověď na tuto skutečnost, napříč mnoha lety co toto odvětví existuje, vznikly jisté vzory, které tyto
-  situace řeší. Říká se jim - Návrhové vzory (Design patterns).
-
-  Návrhové vzory, nejsou nutně vázány na kontrétní jazyk a tudíž je toto téma vhodné i pro ne-pythonisty.
-
-  **POZOR, ZMĚNA TERMÍNU!**
-
-  Omlouváme se, ale z preventivních důvodů jsme se rozhodli změnit termín našeho pravidelného setkání.
-  Předběžně jsme jej posunuli na 13.5. Uvidíme, jak se bude situace vyvíjet.
-  Díky za pochopení.
-  Za organizátory, Tomáš Bedřich
-
+  Sejdeme se alespoň online.
+  Za poslední dva měsíce někteří z nás získli nové zkušenosti, dostali se do nových oblastí případně získali nový úhel
+  pohledu na svět a o tyto zkušenosti bychom se rádi vzájemně podělili.
+  Doufáme, že nejste tématy kolem Covid-19 a videokonferencemi zahlceni natolik, abyste naše setkání zavrhli. 
+  Pro setkání jsme vybrali aplikaci Zoom adresa je: https://1url.cz/@pyvo_plzen_14 Heslo: 111444 
+  Meeting ID: 889 9187 0961
+  V případě problémů napište na mail filip.vaculik@jsi.cz
+  
 talks:
   - title: Úvodní slovo
     speakers:
       - Filip Vaculík
-      - Tomáš Bedřich
-    description: |
-      Pokud máte zkušenost se zajímavou technologii, projektem, nebo chcete prezentovat sebe, či svou firmu a její aktivity,
-      nebojte se přihlásit o slovo. K dispozici můžete mít plných 5 minut pro své téma.
-      Téma je možno přihlásit kdykoliv během akce u libovolného z organizátorů.
-      Také sbíráme náměty na přednášky pro další Pyva viz. [pyvo.cz#info-for-speakers](https://pyvo.cz/#info-for-speakers).
-  - title: Dekorátory
+      - Jan Smitka
+  - title: Pohledem Lyntu
     speakers:
-      - Tomáš Bedřich
+      - Jan Smitka, Lynt services 
     description: |
-      Dekorátor je jednoduchý a užitečný návrhový vzor. Našel si své místo hlavně v srdcích Pythonistů pro svou možnost elegantně ovlivňovat chování funkcí, ale poslední dobou proniká i do jiných jazyků. Pojďme se s ním prakticky seznámit na příkladech.
-  - title: Publisher-subscriber
+      Ve společnosti Lynt nejsme jen vývojáři, jsme především marketingová agentura. Díky tomu máme nadhled i nad businessem našich klientů, proto v pár minutách vypíchnu pár zajímavostí, se kterými se my či naši klienti v této době potýkáme, a pár nástrojů, kterými zefektivňujeme naši práci.
+  - title: Pomocné akce 
     speakers:
-      - Tomáš Bedřich
+      - Filip Vaculík
     description: |
-      Tento systém se dá vysvětlit na jenom slajdu, ale možnosti jeho použití jsou nedozírné. Od zjednodušení komplexních programů až po komunikaci mezi servery po celém světě. Opět se zaměříme hlavně na inspiraci a příklady reálného použití.
-  - title: Jedináčci, továrny a borgové
+      Rád bych vám zprostředkoval pohled člověka, který se vrhnul do pomocných dobrovolnických akcí ve skupině cesko.digital a nejen tam. 
+  - title: 
```

Comment:
Tady chybí název přednášky.

Reply:
Díky za info.
To tam vůbec nemá být.
Já jsem usoudil, že problém je v tom, že tam nemáme místo setkání. 
Jdu se to pokusit opravit.

Reply:
Jsem v tom logu fakt ztracený...
https://travis-ci.org/github/pyvec/pyvo-data/builds/686529991